src/rustup-cli/help.rs

```diff
@@ -78,7 +78,23 @@ inferred, so the above could be written:
     $ rustup default stable-msvc
 
 Toolchain names that don't name a channel instead can be used to name
-custom toolchains with the `rustup toolchain link` command.";
+custom toolchains with the `rustup toolchain link` command. This can
+also be used to symlink toolchains from local builds. For more
+information see `rustup toolchain help link`.";
+
+pub static TOOLCHAIN_LINK_HELP: &'static str =
+r"
+Symlinks a toolchain from a local directory specified by 'path' and
+gives it a custom name specified by 'toolchain' if it does not name a
+standard release channel.
```

Comment:
The call order is `toolchain_link` (in `rustup-cli/rustup-mode.rs`) -> `get_toolchain` (in `rustup/config.rs`) -> `Toolchain::from` (in `rustup/Toolchain.rs`) -> `resolve_toolchain` (in `rustup/config.rs`).